sycl/doc/extensions/supported/sycl_ext_intel_fpga_device_selector.asciidoc

```diff
@@ -90,18 +90,25 @@ supports.
 
 |2
 |fpga_simulator_selector added.
+
+|3
```

Comment:
Sorry for not noticing this sooner.  Did you change the DPC++ header to predefine `SYCL_EXT_INTEL_FPGA_DEVICE_SELECTOR` to `3`?  I don't see a change like that in this PR.

Reply:
Missed that one. It has been incremented!